examples/01_Exact_GPs/Simple_GP_Regression.ipynb

```diff
@@ -274,7 +274,21 @@
     "\n",
     "In the next cell, we make predictions with the model. To do this, we simply put the model and likelihood in eval mode, and call both modules on the test data.\n",
     "\n",
-    "Just as a user defined GP model returns a `MultivariateNormal` containing the prior mean and covariance from forward, a trained GP model in eval mode returns a `MultivariateNormal` containing the posterior mean and covariance. Thus, getting the predictive mean and variance, and then sampling functions from the GP at the given test points could be accomplished with calls like:\n",
+    "Just as a user defined GP model returns a `MultivariateNormal` containing the prior mean and covariance from forward, a trained GP model in eval mode returns a `MultivariateNormal` containing the posterior mean and covariance.\n",
+    "\n",
+    "If we denote a test point (`test_x`) as `x*` with the true output being `y*`, then `model(test_x)` returns the model posterior distribution `p(f* | x*, X, y)`, for training data `X, y`. This posterior is the distribution over the function we are trying to model, and thus quantifies our model uncertainty.\n",
+    "\n",
+    "In contrast, `likelihood(model(test_x))` gives us the posterior predictive distribution `p(y* | x*, X, y)` which is the probability distribution over the predicted output value. This includes the _likelihood noise_ which is the noise in your observation (e.g. due to noisy sensor). Recall in our problem setup \n",
+    "\n",
+    "$$\n",
+    "\\begin{align}\n",
+    "y &= \\sin(2\\pi x) + \\epsilon\n",
+    "\\end{align}\n",
+    "$$\n",
+    "\n",
+    "where 𝜖 is the likelihood noise for each observation. Since the posterior predictive distribution includes the likelihood noise, it will provide better prediction performance.\n",
```

Comment:
> Since the posterior predictive distribution includes the likelihood noise, it will provide better prediction performance

This seems like an odd phrasing here - what's "better"? I think we're just providing predictions for two different things - over the latent function value vs. over the observations. Maybe we can rephrase this here / be more precise about this?

Reply:
@Balandat I was paraphrasing @gpleiss. I'll update this.